formats/cbor/commonTest/src/kotlinx/serialization/PolymorphismTestData.kt

```diff
@@ -6,38 +6,6 @@ package kotlinx.serialization
 
 import kotlinx.serialization.modules.*
 
-@Serializable
-open class PolyBase(val id: Int) {
```

Comment:
where did these go? I can't find them in changed files

Reply:
These were no longer used, so I deleted them